packages/atoms/src/Button/IconButton.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { TouchableOpacity } from 'react-native';
+
+import { Icon, IconName } from '@trezor/icons';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { ButtonColorScheme, ButtonSize } from './Button';
+
+type Props = {
+    iconName: IconName;
+    onPress: () => void;
+    isRounded?: boolean;
+    colorScheme?: ButtonColorScheme;
+    size?: ButtonSize;
+};
+
+type StyleProps = {
+    isRounded: boolean;
+    size: ButtonSize;
+    colorScheme: ButtonColorScheme;
+};
+
+const iconButtonStyle = prepareNativeStyle<StyleProps>(
+    (utils, { isRounded, colorScheme, size }) => {
+        const colorSchemeStyles = {
```

Comment:
we should have also transparent button probably, if you check #5357 you it's that finger print button

Reply:
I thought this was meant to be same as button schemes and NumPadButton is a specific cmp? But I can add it, just making sure it's needed somewhere else apart from num pad button where it's not icon as a content.